src/SimpleWallet/SimpleWallet.cpp

```diff
@@ -3291,6 +3292,8 @@ int main(int argc, char *argv[])
     _CrtSetDbgFlag(_CRTDBG_ALLOC_MEM_DF | _CRTDBG_LEAK_CHECK_DF);
 #endif
 
+    Qwertycoin::Breakpad::ExceptionHandler exceptionHandler;
```

Comment:
Also would be wary about linking this to Wallet and PaymentGate... from a password security standpoint for users. Maybe debug builds only?

Reply:
I think we should keep this `ON` for both Debug and Release builds.

But for security reasons we should never upload those dumps from user machines to anywhere on the internet.

Supporting breakpad in release builds will helps us get more information from users when they post bug reports on GitHub. Basically, we will guide user through readable dump generation process and ask them to post only a stack trace in order to figure out on what like application did actually crash.

Reply:
> I think we should keep this `ON` for both Debug and Release builds.
> 
> But for security reasons we should **never automatically upload** those dumps from user machines to anywhere on the internet.

Didn't realize that automatic uploads weren't enabled by default.  All good on that concern, then.  Thanks for clarifying